easy-ui-react/src/Menu/Menu.module.scss

```diff
@@ -19,6 +19,12 @@
   @include Menu.menuList;
 }
 
+.sectionTitle {
+  color: theme-token("color.primary.900");
+  font-weight: design-token("font.weight.semibold");
+  padding: 0 component-token("menu", "padding.x");
+}
```

Comment:
looks as though the Figma has the typography for the title as s1, which admittedly is a bit cryptic, but i believe it roughly maps to the `subtitle1` font style in `easy-ui-tokens/src/font/styles`

![Screenshot 2024-11-06 at 5 11 40 PM](https://github.com/user-attachments/assets/2fc07408-a600-46b9-a2a8-651d67b06f1a)

you can achieve this in a couple ways, either by using the `Text` component

```
<Text variant="subtitle1" color="primary.900">text</Text>
```
or through the font-styles mixin helper which taking your code as an example, it would become

```
.sectionTitle {
  @include font-style("subtitle1");
  color: theme-token("color.primary.900");
  padding: 0 component-token("menu", "padding.x");
}
```